checks.d/http_check.py

```diff
@@ -161,10 +164,15 @@ def __init__(self, name, init_config, agentConfig, instances):
                     user=proxy_settings['user'],
                     password=proxy_settings['password'],
                     uri=uri)
-            self.proxies = {
-                'http': "http://{uri}".format(uri=uri),
-                'https': "https://{uri}".format(uri=uri)
-            }
+            self.proxies['http'] = "http://{uri}".format(uri=uri)
+            self.proxies['https'] = "https://{uri}".format(uri=uri)
+        else:
+            self.proxies['http'] = environ.get('http', None)
+            self.proxies['https'] = environ.get('https', None)
```

Comment:
Good catch! That was my bad, indeed the env variables on *nix systems have `_PROXY` appended. Fixing!